homeassistant/components/blueprint/schemas.py

```diff
@@ -67,6 +69,21 @@ def is_blueprint_instance_config(config: Any) -> bool:
     }
 )
 
+BLUEPRINT_INPUT_SECTION_SCHEMA = vol.Schema(
+    {
+        vol.Optional(CONF_NAME): str,
+        vol.Optional(CONF_ICON): str,
+        vol.Optional(CONF_DESCRIPTION): str,
+        vol.Optional(CONF_COLLAPSED): bool,
+        vol.Required(CONF_INPUT, default=dict): {
+            str: vol.Any(
+                None,
+                BLUEPRINT_INPUT_SCHEMA,
+            )
+        },
+    }
+)
+
 BLUEPRINT_SCHEMA = vol.Schema(
```

Comment:
I think this needs a custom validator which fails if the input keys are not unique.

Reply:
Added validator.